### PR TITLE
RFP: filecoin-block-explorer by IPFSUNION Team

### PR DESCRIPTION
**Pull request by maobisheng**
Our team combined the project requirements and our own design philosophy to design a fully functional, interactive and friendly filecoin block explorer.

### CODE REVIEW

**Comment by maobisheng** [comment]
https://github.com/maobisheng/devgrants/blob/master/open-grant-proposals/rfp-filecoin-block-explorer.md

File: open-grant-proposals/rfp-filecoin-block-explorer.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Total Funding Amount: $1500
+
+Milestone: 
+
+| Milestone No. | Milestone Description | Funding | Estimated Timeframe |
+| --- | --- | --- | --- |
+| 1 | High-fidelity design mockups, running staging site with some UI elements implemented | $400 | 2 weeks |
+| 2 | Completed implementation of block explorer | $800 | 4 weeks |
+| 3 | All project deliverables are completed and added to the [Filecoin Shipyard](https://github.com/filecoin-shipyard) (website, documentation, codebase) | $300 | 2 weeks |
+
```
Comment (eshon) [inline open-grant-proposals/rfp-filecoin-block-explorer.md]: Hi @IPFSUNION,

Some questions about your dev grant.

1. The amount requested seems really small. What specifically do you need a dev grant for?

2. above it says "the version V1.0 in March next year" - to confirm, this means March 2020 right?

Comment (maobisheng) [reply open-grant-proposals/rfp-filecoin-block-explorer.md]: Some questions about your dev grant proposal.
The amount requested seems really small. What specifically do you need a dev grant for?
——Before we apply the devgrant, we have already started the filscan project in the Oct of 2019, we just want some bonus for it. 

above it says "the version V1.0 in March next year" - to confirm, this means March 2020 right?
——Yes, we have almost finished it when we submit the devgrant last year, and we will open the url "filscan.in" in the coming two weeks.

Comment (eshon) [reply open-grant-proposals/rfp-filecoin-block-explorer.md]: @maobisheng - filscan.io was dev granted to a group in wave 1. Is it possible for this to have a different name? e.g. filscout?

Comment (maobisheng) [reply open-grant-proposals/rfp-filecoin-block-explorer.md]: > @maobisheng - filscan.io was dev granted to a group in wave 1. Is it possible for this to have a different name? e.g. filscout?

No problem.

**Comment by eshon** [comment]
@maobisheng - What is your backend in? What type of database?

**Comment by maobisheng** [comment]
> @maobisheng - What is your backend in? What type of database?

Golang + MySQL + Redis

**Comment by maobisheng** [comment]
https://filscout.io go live now, the first Filecoin Explorer which can run on both PC and mobile, with multi-language(English & Chinese) support.

**Comment by eshon** [comment]
Hi @maobisheng,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal!

We would like to award a dev grant for the Filscout.io Block Explorer. 🎉 Congratulations!

Please note that:

请注意，合同不允许将收到开发者津贴用作其他不相关产品或服务的促销，并且不能宣传为合作伙伴企业。

Acceptance also means that your code will need to be open sourced to https://github.com/filecoin-shipyard.

As an open source project and live community block explorer we may also ask for bug fixes and make pull requests for improvements to the UI. 

Our DevGrants team will be in touch soon about next steps.

**Pull request closed without merging**